components/tikv_util/src/yatp_pool/future_pool.rs

```diff
@@ -472,10 +472,10 @@ mod tests {
         // full
         assert!(spawn_long_time_future(&read_pool, 8, 100).is_err());
 
-        assert!(rx.recv().is_ok());
-        assert!(rx.recv().is_ok());
-        assert!(rx.recv().is_ok());
-        assert!(rx.recv().is_ok());
+        rx.recv().unwrap().unwrap();
```

Comment:
Why unwrap twice?

Reply:
I think it should check the task succeed.